### DISCUSSION TITLE
Cobbler EPEL 9 versions

### DISCUSSION THREAD

**Discussion by pauliopolis** [cobbler/cobbler#3543] [Q&A] [answered]
Older cobbler versions now appeared in EPEL 9 - ?? at least that's what I assume they are
![image](https://github.com/cobbler/cobbler/assets/14337141/137aa190-3706-47af-8b7a-1780e36245c2)

**Comment by opoplawski** [comment, marked as answer]
Yes, cobbler3.2 is now available in EPEL9.
